INN/umbrella-citylimits
# Survey styling

## Changes

This pull request makes the following changes:

- Adds styling for surveys on the special projects template

![screencapture-citylimits-test-series-the-class-of-2013-2019-11-11-14_20_55](https://user-images.githubusercontent.com/18353636/68614849-c464db80-048f-11ea-8057-36ef689c23d1.png)

## Why



For #72

## Testing/Questions

Features that this PR affects:

- Gravity form widgets placed on special projects where the form contains the class `special-projects-survey`


Questions that need to be answered before merging:

- [ ] What other elements should we style?
- [ ] Should the textarea be the same width as the normal input? If so, the right side of the form will look very empty and bare. Is that ok?

Steps to test this PR:

1. Modify an existing GF survey (I used ID #6 Job Survey)
2. Under "Form Settings" add `special-projects-survey` to the "CSS Class Name" field
3. Add in different types of form fields
4. Add the Gravity Form widget to a special projects widget area
   - Title: empty
   - Select a form: The one you edited above
   - Display form title: yes
   - Display description: yes
